src/scss/_action.scss

```diff
@@ -3,12 +3,19 @@
   margin: 0;
   padding: 0;
   height: 2.75em;
-  position: absolute;
   z-index: 10;
+  position: fixed;
+  top: 0;
+
+  &__jsload {
+    position: absolute;
+    top: auto;
+  }
 
   &__link {
     @extend .type__link;
-    height: 2.75em;
+    height: 52px;
```

Comment:
There are a few times where you use px for height and top, so those could be ems instead.